webpack.config.js

```diff
@@ -28,15 +28,18 @@ const config = {
       path.resolve('./node_modules'),
     ],
   },
+  devtool: 'eval-source-map',
   module: {
+    // We need to load flexboxgrid with css-modules, but others need to be loaded
+    // with regular css loader.
     loaders: [{
       test: /\.css$/,
       loader: 'style-loader!css-loader?modules&importLoaders=1&localIdentName=[name]__[local]___[hash:base64:5]!postcss-loader',
-      exclude: /node_modules/,
+      exclude: /node_modules\/?=flexboxgrid/,
```

Comment:
we can do this for now, but I'm not sure how maintainable this will be in the future (what if we find another dependency that needs this?)